lms/static/sass/course/courseware/_courseware.scss

```diff
@@ -191,6 +191,14 @@ div.course-wrapper {
         margin: 0 auto;
       }
     }
+    .disable-proctored-exam-btn {
+        i {
```

Comment:
NIT: can we use a class here instead of an element selector? Using the .icon class instead of the selector would be preferred.